ambry-api/src/main/java/com/github/ambry/clustermap/ReplicaStatusDelegate.java

```diff
@@ -37,6 +38,17 @@ public boolean seal(ReplicaId replicaId) {
     return clusterParticipant.setReplicaSealedState(replicaId, true);
   }
 
+  /**
+   * Sets replicaId to partially sealed status. Replicas marked as partially sealed will not be able to take writes for
+   * metadata chunks whose id is reserved at the start of the upload.
```

Comment:
nit: please fix the comments,
1. don't need relate it to  metadata chunks design
2. s/read-only/xxxx
3. s/sealed/partially sealed.

Reply:
fixed.